browser/plugins/network-breadcrumbs.js

```diff
@@ -0,0 +1,144 @@
+/* global XMLHttpRequest */
+const breadcrumbType = 'network'
+let restoreFunctions = []
+let client
+
+/*
+ * Leaves breadcrumbs when network requests occur
+ */
+module.exports = {
+  init: (_client) => {
+    client = _client
+    monkeyPatchXMLHttpRequest()
+    monkeyPatchFetch()
+  },
+  destroy: () => {
+    restoreFunctions.forEach(restore => restore())
```

Comment:
I really want to find a better way of doing this. We basically only need these destroy methods for tests, so I want a way to have them compile out of the production build. Don't worry about addressing this I just wanted to raise it as it's been bugging be for a bit.